core/modules/ccontrol/UserQuerySelect.cc

```diff
@@ -248,11 +247,7 @@ QueryState UserQuerySelect::join() {
     bool successful = _executive->join(); // Wait for all data
```

Comment:
As long as 'uq->submit();` precedes `uq->join();`, the not submitted case would never be reached and isn't terribly meaningful. The the query was cancelled and why is probably the important information, but we aren't sending that back to the user as we usually have some valid rows to send back.